src/main/java/net/mcreator/blockly/java/ProcedureCodeOptimizer.java

```diff
@@ -55,6 +55,10 @@ else if (code.startsWith("/*@ItemStack*/")) {
 			prefix = "/*@ItemStack*/";
 			toClean = toClean.substring(14);
 		}
+		else if (code.startsWith("/*@int*/")) {

```

Comment:
This line should be with the previous one.

Reply:
What do you mean? If it's indentation, it's fine to me

Reply:
It should be `} else if (condition)`.